op-batcher/flags/throttle_flags.go

```diff
@@ -0,0 +1,179 @@
+package flags
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	// Block-builder side
+	DefaultThrottleTxSizeLowerLimit    = 150
+	DefaultThrottleTxSizeUpperLimit    = 10_000
+	DefaultThrottleBlockSizeLowerLimit = 21_000
+	DefaultThrottleBlockSizeUpperLimit = 130_000_000
```

Comment:
The old limit was `130_000`, why 1000x larger now?

Reply:
Good catch, fixed here [3d6f521](https://github.com/ethereum-optimism/optimism/pull/17032/commits/3d6f5217b5aa4ecec04c942b20ea8c34cbfa5601)